<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="OCL0181414" name="Test" tests="2" failures="1" timestamp="2 Apr 2021 04:35:01 GMT" time="38.718" errors="0">
  <testcase name="checkIphonePrice" time="11.933" classname="testngrecap.paracticeTNG1">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [509.00] but found [673.99]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [509.00] but found [673.99]
	at testngrecap.paracticeTNG1.checkIphonePrice(paracticeTNG1.java:45)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- checkIphonePrice -->
  <testcase name="clickOnSell" time="6.654" classname="testngrecap.paracticeTNG2"/>
</testsuite> <!-- Test -->
